### ISSUE TITLE
EmptyFunctionBlock should flag empty functions defined inside other functions

### ISSUE THREAD

**Issue by egor-n** [arturbosch/detekt#998]
## Expected Behavior
Functions defined inside functions should follow the same rules.
```kotlin
fun a() {
    fun b() {}
}
```

`fun b()` should be reported as an empty function.

## Current Behavior
```kotlin
// EmptyFunctionBlockSpec.kt

given("an empty function inside a another function") {
	it("should flag the empty function") {
		val code = """
		private class A {
			fun a() {
				fun b() {}
			}
		}
		"""
		assertThat(subject.lint(code)).hasSize(1)
	}
}
```

This test case fails.
